docs/documentation.md

```diff
@@ -0,0 +1,20 @@
+# [Port Scanner](https://vinitshahdeo.github.io/PortScanner/) in Software Defined Networking
+
+A port scanner is an application intended to test a server or host for open ports. Such an application might be utilized by managers to check security approaches of their systems and by assailants to recognize arrange administrations running on a host and adventure vulnerabilities.
+
+It ought to be noticed that port scanning can be viewed as, or interpreted as, a wrongdoing. One ought to never execute a port scanner against any site or IP address without unequivocal, composed, authorization from the proprietor of the server or PC that you are focusing on. Port scanning resembles to going into somebody's home and looking at all of their entryways and windows. There is extremely just motivation behind why anybody would do this, and it is to survey protections and vulnerabilities. In this way, on the off chance that you have no rhyme or reason to test these things, it tends to be expected you are a criminal.
+
+**Software-Defined Networking (SDN)** technology is being used for efficient and real-time defense against cyber attacks.
+With this port scanner, I just attempt to connect at various ports. If I'm able to connect to open ports, then I know at least the port is open. The code is in python and the server is built using express.js. 
+
+
+## Working
+
+The server code is executed first where it opens 10 random ports (random numbers generated by `randomNum.js`) and the `index.js` starts ten servers. Then the `scanner.py` searches for all the open ports on remote server using sockets (socket library of python). The `index.js` is coded in express which is a server side scripting language and creates and starts servers. The scanning code for open ports is build using python programming language. The `scanner.py` code imports and uses several python libraries like socket, subprocess and sys. 
+`index.js` imports randNum.js to generate random number and open ports at those `localhost`. It also imports config.json which stores values of low and high i.e. values within whose range the ports will open. In this code, it is set from 1 (inclusive) to 80 (exclusive). The code generates 10 random numbers from 1 to 80 i.e low to high and starts servers at those ports. 
```

Comment:
The code generates 10 random numbers from **1 to 80** i.e low to high and starts servers at those ports. 

Should be changed to

The code generates 10 random numbers from **1 to 8888** i.e low to high and starts servers at those ports.